bears/general/CasingBear.py

```diff
@@ -0,0 +1,134 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.naming_conventions import (
+    to_camelcase, to_pascalcase, to_snakecase)
+from bears.general.AnnotationBear import AnnotationBear
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.results.SourceRange import SourceRange
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class CasingBear(LocalBear):
+
+    def run(self,
+            filename,
+            file,
+            casing: str,
+            language: str):
+        """
+        Checks whether all identifier names (variables, classes, objects)
+        follow a certain naming convention.
+
+        :param casing:   camelCasing or snake_casing or PascalCasing
```

Comment:
Well this doesn't apply just to variable names: function names, class names, .... But I'm open to suggestions - here or there?